pkgs/top-level/all-packages.nix

```
@@ -4168,6 +4168,8 @@ with pkgs;
 
   cwebbin = callPackage ../development/tools/misc/cwebbin { };
 
+  cve-bin-tool = callPackage ../tools/security/cve-bin-tool { };
```

```suggestion
  cve-bin-tool = with python3Packages; callPackage ../tools/security/cve-bin-tool { };
```

---

No need to use `with`.

```suggestion
  cve-bin-tool = python3Packages.callPackage ../tools/security/cve-bin-tool { };
```